core/src/main/scala/io/snappydata/SnappyTableStatsProviderService.scala

```diff
@@ -164,18 +164,24 @@ object SnappyEmbeddedTableStatsProviderService extends TableStatsProviderService
 
   override def getStatsFromAllServers(sc: Option[SparkContext] = None): (Seq[SnappyRegionStats],
       Seq[SnappyIndexStats], Seq[SnappyExternalTableStats]) = {
+    val currTableSizeInfo = tableSizeInfo.values.toSeq
     var result = new java.util.ArrayList[SnappyRegionStatsCollectorResult]().asScala
     var externalTables = scala.collection.mutable.Buffer.empty[SnappyExternalTableStats]
     val dataServers = GfxdMessage.getAllDataStores
     try {
       if (dataServers != null && dataServers.size() > 0) {
         result = FunctionService.onMembers(dataServers)
-            .withCollector(new GfxdListResultCollector())
-            .execute(SnappyRegionStatsCollectorFunction.ID).getResult().
+            // .withCollector(new GfxdListResultCollector())
```

Comment:
Why is this commented? I don't know much about function execution, but was there any limitation?

Reply:
Yes, getResult with timeout is not expected to be invoked for GemFireXD.